server/lib/routes.js

```diff
@@ -30,7 +30,10 @@ module.exports = function (config, i18n) {
     require('./routes/get-client.json')(i18n),
     require('./routes/post-metrics')(),
     require('./routes/post-csp')({
-      path: config.get('csp.reportUri')
+      path: config.get('csp.reportUri'),
+      write: function (entry) {
```

Comment:
@shane-tomlinson this looks a bit strange here, maybe process.write is default and there is an option to pass your own `.write` instead?